kong/db/strategies/postgres/connector.lua

```diff
@@ -552,6 +553,11 @@ function _mt:query(sql, operation)
 
   res, err, partial, num_queries = conn:query(sql)
 
+  if (kong and kong.configuration and kong.configuration.log_level == "debug")
+          and in_admin_api and operation == "write"
+          and res and res[1] and res[1]._pg_transaction_id then
+    kong.response.set_header('X-Kong-Test-Transaction-ID', res[1]._pg_transaction_id)
```

Comment:
too long judgment condition, can it be turned into a local variable?

Reply:
A local variable that is used exactly once is not better.